client/core/helpers.go

```diff
@@ -179,7 +179,7 @@ func (ord *OrderReader) SideString() string {
 
 func (ord *OrderReader) percent(filter func(match *Match) bool) string {
 	var sum uint64
-	if ord.Sell {
+	if ord.Sell || ord.IsMarketBuy() {
```

Comment:
Are `FilledFrom` and `FilledTo` OK?  They both call `sumTo`/`sumFrom`.  Should this `IsMarketBuy` condition be in the `sum` functions instead?